website/static/files/build_orca_inventory_refined.sql

```diff
@@ -67,6 +67,7 @@ BEGIN
     , cumulus_granule_id  text NOT NULL
     , execution_id  	  text NOT NULL
     , ingest_time         timestamp with time zone NOT NULL
+    , cumulus_createdAt    timestamp with time zone NOT NULL
```

Comment:
See comment on name and description. This should match any changes made in orca_sql.py.